apps/cli/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@webstudio-is/tsconfig/base.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "customConditions": null,
+    "incremental": true
+  }
```

Comment:
Do we need any of these options?

Reply:
We don't need them, we just need `compilerOptions.module` to be set to `module` to support nodejs native `import` assertions for json improts.

Reply:
could this be a https://github.com/webstudio-is/webstudio-builder/tree/main/packages/tsconfig src/node ? so we have a standard ts config for all node native scripts?